lib/jasmine-core/jasmine.js

```diff
@@ -452,6 +453,46 @@ if (typeof window == void 0 && typeof exports == 'object') {
   exports.Spec = jasmineRequire.Spec;
 }
 
+/*jshint bitwise: false*/
+
+getJasmineRequireObj().Order = function() {
+  function Order(options) {
+    this.random = 'random' in options ? options.random : true;
+    var seed = this.seed = options.seed || generateSeed();
+    this.sort = this.random ? randomOrder : naturalOrder;
+
+    function naturalOrder(items) {
+      return items;
+    }
+
+    function randomOrder(items) {
+      items.sort(function(item) {
+        return jenkins(seed + item.id);
+      });
+    }
+
+    function generateSeed() {
+      return String(Math.random()).slice(-5);
+    }
+
+    function jenkins(key) {
```

Comment:
I had to go look up why this would just be called `jenkins` to find the Bob Jenkins algorithm. It would be nice to have something a bit more descriptive